### PR TITLE
readme: update from the slides from the presentation

### PR DESCRIPTION
**Pull request by lavocatt**
From https://lavocatt.github.io/manifest-db-slides/#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,123 @@ Manifest-db
...
 
-### Update workflow
+# Introduction
 
-#### 1/ on GitHub: trigger the Gitlab CI
+![](https://mermaid.ink/svg/pako:eNo9j70OwjAMhF_F8kxfoAMTDJXoAiPpYBqnWGoclDpCCPHuhN_NJ393p7vjmDxji2FO1_FM2WC3d9odu0gTg-cgKiZJB2iaiRUiqQRerGnW0B_7rxqc9hWAtJyKzB5e3y5-Q0RDGnCFkXMk8bXs7hTAoZ05ssO2nrWIymwOnT4qWi6ejLdeLGVsA80Lr5CKpcNNR2wtF_5BG6EpU_xT_Db1n1XvcY8nBzNOUA)
+
+## Image definitions
+
+Image definitions are the Way we describe an image and do exist for every image
```
Comment (ochosi) [inline README.md]:
```suggestion
Image definitions are the way we describe an image and there is one for every image
```

File: README.md [hunk 2]
```diff
@@ -3,10 +3,123 @@ Manifest-db
...
+we support. Definitions can be found in
+[osbuild-composer/internal/distros](https://github.com/osbuild/osbuild-composer/tree/main/internal/distro)
+
+### Image definitions must be tested
+
+- We need to run OSBuild against every one of them
```
Comment (ochosi) [inline README.md]:
```suggestion
- We need to run OSBuild against each one of them
```

File: README.md [hunk 3]
```diff
@@ -3,10 +3,123 @@ Manifest-db
...
+- Check the build is *correct*
```
Comment (ochosi) [inline README.md]:
```suggestion
- Check whether the build is *correct*
```

File: README.md [hunk 4]
```diff
@@ -3,10 +3,123 @@ Manifest-db
...
+
+* Generate an image for each manifest
+* Generate an image info for each image
+
+Generation is a [heavy duty
+task](https://github.com/osbuild/manifest-db/blob/main/.gitlab-ci.yml#L75):
 
+- Requires a machine per `arch+distro`
+- And up to
+[2h10](https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db/-/jobs/3119412435) to build a set of manifests on a machine
```
Comment (ochosi) [inline README.md]: This link is bound to expire soon, no?

Comment (lavocatt) [reply README.md]: I'm trying to find out what's the gitlab policy regarding the storage lifecycle. I don't find any data on the subject. There's only timeout related information for CI's artifacts.

**Comment by ochosi** [comment]
I think the content from the presentation makes sense in the README. However, I would expect it to be structured more like a README.
So basically start with what the thing in the repo is and what it does before getting into the weeds of defining terms.
Finally, the - from my pov - most important bit about the update workflow is currently at the bottom.

Just a very quick and dirty suggestion for a structure:

1. Introduction/Summary of what manifest-db is and does
2. Update cycle of manifest-db
3. Which problem does manifest-db solve (and what did we do before)
4. Appendix: Terminology
4.1 Manifests
4.2 Image Info
...

**Review by ochosi — CHANGES_REQUESTED** [review]
See my comment about the structure of the README

**Comment by lavocatt** [comment]
> See my comment about the structure of the README

Thanks for the comments! I think it's quite better now :smile:

File: README.md (hunk 4 above)
Comment (lavocatt) [reply README.md]: I removed it in the end.

**Comment by ochosi** [comment]
@lavocatt Thanks for the work you put into this - it's so much better :heart: 

There's only one more thing I can offer, which is language improvement suggestions. Let me know if you want me to review the text from that angle and add suggestions to it. Otherwise it's good to go from where I stand!

**Review by ochosi — APPROVED** [review]

**Comment by lavocatt** [comment]
> @lavocatt Thanks for the work you put into this - it's so much better heart
> 
> There's only one more thing I can offer, which is language improvement suggestions. Let me know if you want me to review the text from that angle and add suggestions to it. Otherwise it's good to go from where I stand!

Thanks Simon for taking the time to review! Yes I would love to see your suggestions.

**Pull request merged**